mkvcore/blockreader.go

```diff
@@ -0,0 +1,133 @@
+// Copyright 2019 The ebml-go authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package mkvcore
+
+import (
+	"fmt"
+	"io"
+	"reflect"
+	"sync"
+
+	"github.com/at-wat/ebml-go"
+)
+
+type ReadableBlock struct {
+	Header      interface{}
```

Comment:
I guess this is for reading `TrackEntry`.
it would be better to add an accessor method to `BlockReader` instead of accessing member variable directly.
https://github.com/at-wat/ebml-go/blob/5591cbaca74dff5f55fde48e069ac68ed8bc8702/mkvcore/interface.go#L24-L28

Then, `ReadableBlock struct` can be `blockReader struct`.